### PR TITLE
Clash errors (Draft)

### PR DESCRIPTION
**Pull request by amityadav0**

### CODE REVIEW

File: packages/algob/src/builtin-tasks/run.ts [hunk 1]
```diff
@@ -55,11 +55,10 @@ export function splitAfter (
...
 function loadCheckpointsIntoCPData (cpData: CheckpointRepo, scriptPaths: string[]): CheckpointRepo {
-  return scriptPaths
-    .map(loadCheckpoint)
-    .reduce(
-      (out: CheckpointRepo, checkpoints: Checkpoints) => out.merge(checkpoints),
-      cpData);
+  for (const s of scriptPaths) {
+    cpData.merge(loadCheckpoint(s), s);
+  }
+  return cpData;
```
Comment (robert-zaremba) [inline packages/algob/src/builtin-tasks/run.ts:61]: :+1:

File: packages/algob/src/lib/script-checkpoints.ts [hunk 2]
```diff
@@ -81,23 +83,43 @@ export class CheckpointRepoImpl implements CheckpointRepo {
...
+  mergeToGlobal (cp: Checkpoints, scriptName: string): CheckpointRepo {
+    this.allCPs = this._mergeTo(this.allCPs, cp, this.scriptMap);
+
+    if (scriptName.charAt(0) === 'a') {
+      scriptName = scriptName.slice(10, scriptName.length - 8);
+    }
+
+    const keys: string[] = Object.keys(cp);
+    for (let i = 0; i < keys.length; i++) {
+      const orig = cp[keys[i]];
```
Comment (robert-zaremba) [inline packages/algob/src/lib/script-checkpoints.ts:111]: You can use:
```
for (let k of keys) { .... }
```
instead the two lines above.

**Review by robert-zaremba — APPROVED** [review]
LGTM

File: packages/algob/src/lib/script-checkpoints.ts (hunk 2 above)
Comment (amityadav0) [reply packages/algob/src/lib/script-checkpoints.ts:111]: ok

**Pull request closed without merging**